### PR TITLE
Extract deployment information

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: universal-login-react/src/ui/Onboarding/Onboarding.tsx [hunk 1]
```diff
@@ -7,6 +7,8 @@ import {getStyleForTopLevelComponent} from '../../core/utils/getStyleForTopLevel
 import {ReactModalContext, ReactModalProps, ReactModalType, TopUpProps} from '../../core/models/ReactModalContext';
 import {useModalService} from '../../core/services/useModalService';
 
+export const DEPLOYMENT_INFO = 'It takes time to register your username and deploy your wallet. In order to do so, we need to create a transaction and wait until the Ethereum blockchain validates it...';
```
Comment (xitronix) [inline universal-login-react/src/ui/Onboarding/Onboarding.tsx]: move to core/constants?

File: universal-login-wallet/src/ui/react/Modals/Modal.tsx [hunk 2]
```diff
@@ -40,6 +40,7 @@ const Modal = () => {
             action={'Wallet creation'}
             relayerConfig={relayerConfig!}
             className={'jarvis-waiting-for-deploy'}
+            info={DEPLOYMENT_INFO}
```
Comment (rzadp) [inline universal-login-wallet/src/ui/react/Modals/Modal.tsx]: This param is missing in `case 'waitingForTransfer'`

File: universal-login-react/src/ui/commons/WaitingForTransaction.tsx [hunk 3]
```diff
@@ -8,9 +8,10 @@ import {getStyleForTopLevelComponent} from '../../core/utils/getStyleForTopLevel
 export interface WaitingForTransactionProps extends WaitingForProps {
   relayerConfig: PublicRelayerConfig;
   transactionHash?: string;
+  info?: string;
```
Comment (rzadp) [inline universal-login-react/src/ui/commons/WaitingForTransaction.tsx]: Debatable idea - Maybe make it required? If nothing is to be displayed, then just pass empty string `''`

File: universal-login-react/src/ui/Onboarding/Onboarding.tsx (hunk 1 above)
Comment (rzadp) [reply universal-login-react/src/ui/Onboarding/Onboarding.tsx]: Do we have other strings like this in core/constants?

File: universal-login-react/src/ui/commons/WaitingForTransaction.tsx [hunk 4]
```diff
@@ -19,18 +20,30 @@ const renderWaitingForTransaction = ({action, relayerConfig, transactionHash, ch
           <h3 className="transaction-hash-title">Transaction hash</h3>
           <ExplorerLink chainName={relayerConfig.chainSpec.name} transactionHash={transactionHash} />
         </div>
-        <p className="info-text">It takes time to register your username and deploy your wallet. In order to do so, we need to create a transaction and wait until the Ethereum blockchain validates it...</p>
+        <p className="info-text">{info}</p>
```
Comment (Marik-D) [inline universal-login-react/src/ui/commons/WaitingForTransaction.tsx]:
```suggestion
        {info && <p className="info-text">{info}</p>}
```

File: universal-login-react/src/ui/commons/WaitingForTransaction.tsx [hunk 5]
```diff
@@ -19,18 +20,30 @@ const renderWaitingForTransaction = ({action, relayerConfig, transactionHash, ch
...
+          : <Spinner className="waiting-for-spinner" />
+        }
       </div>
     </div>
   );
 };
+
+const DEPLOYMENT_INFO = 'It takes time to register your username and deploy your wallet. In order to do so, we need to create a transaction and wait until the Ethereum blockchain validates it...';
+
+export const WaitingForDeployment = (props: WaitingForTransactionProps) =>
```
Comment (Marik-D) [inline universal-login-react/src/ui/commons/WaitingForTransaction.tsx]:
```suggestion
export const WaitingForDeployment = (props: Omit<WaitingForTransactionProps, 'action' | 'info'>) =>
```

File: universal-login-react/src/ui/commons/WaitingForTransaction.tsx [hunk 6]
```diff
@@ -19,18 +20,30 @@ const renderWaitingForTransaction = ({action, relayerConfig, transactionHash, ch
...
+  (<WaitingForTransaction
+    action={'Wallet creation'}
```
Comment (Marik-D) [inline universal-login-react/src/ui/commons/WaitingForTransaction.tsx]:
```suggestion
    action="Wallet creation"
```

**Pull request merged**